plugin-dev/Source/Sentry/Private/GenericPlatform/GenericPlatformSentrySubsystem.cpp

```diff
@@ -381,6 +413,27 @@ void FGenericPlatformSentrySubsystem::ClearBreadcrumbs()
 	// Not implemented in sentry-native
 }
 
+void FGenericPlatformSentrySubsystem::AddAttachment(TSharedPtr<ISentryAttachment> attachment)
+{
+	if (!attachment->GetPath().IsEmpty())
+	{
+		AddFileAttachment(attachment);
+	}
+	else
+	{
+		AddByteAttachment(attachment);
+	}
+}
+
+void FGenericPlatformSentrySubsystem::RemoveAttachment(TSharedPtr<ISentryAttachment> attachment)
+{
+	TSharedPtr<FGenericPlatformSentryAttachment> platformAttachment = StaticCastSharedPtr<FGenericPlatformSentryAttachment>(attachment);
+
+	sentry_attachment_t* nativeAttachment = platformAttachment->GetNativeObject();
+
+	sentry_remove_attachment(nativeAttachment);
```

Comment:
If I understand correctly, `FGenericPlatformSentryAttachment` outlives the `sentry_native_t` instance here. Maybe we could reset the native object reference just to be safe and avoid accidentally dereferencing a dangling pointer somewhere down the line?